pysensu_yelp/__init__.py

```diff
@@ -210,3 +212,28 @@ def send_event(
         sock.sendall(json_hash + '\n')
     finally:
         sock.close()
+
+
+def do_command_wrapper():
+    parser = argparse.ArgumentParser(description='Execute a nagios plugin and report the results to a local Sensu agent')
+    parser.add_argument('sensu_dict')
+    parser.add_argument('command', nargs=argparse.REMAINDER)
+    args = parser.parse_args()
+
+    sensu_dict = json.loads(args.sensu_dict)
+
+    p = subprocess.Popen(args.command,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.STDOUT)
+    output, _ = p.communicate()
+    status = p.wait()
+
+    if status > 3:
+        status = 3
+
+    sensu_dict['status'] = status
+    sensu_dict['output'] = output[:1200]
```

Comment:
1200 "sounds" small to me. Sensu doesn't limit it, any particular reason why we need to limit it too much here?
https://github.com/sensu/sensu/blob/6df74023f74f67077583e96088a1ecf8ad0651de/lib/sensu/client/process.rb#L151